public/app/plugins/panel/table/module.tsx

```diff
@@ -75,10 +75,82 @@ export const plugin = new PanelPlugin<PanelOptions, PanelFieldConfig>(TablePanel
     },
   })
   .setPanelOptions((builder) => {
-    builder.addBooleanSwitch({
-      path: 'showHeader',
-      name: 'Show header',
-      description: "To display table's header or not to display",
-      defaultValue: defaultPanelOptions.showHeader,
-    });
+    builder
+      .addBooleanSwitch({
+        path: 'showHeader',
+        name: 'Show header',
+        description: "To display table's header or not to display",
+        defaultValue: defaultPanelOptions.showHeader,
+      })
+      .addRadio({
+        category: ['Table footer'],
+        path: 'footerMode',
+        name: 'Mode',
+        settings: {
+          options: [
+            { label: 'none', value: 'none' },
+            // { label: 'auto', value: 'auto' },  // TODO: when frame meta supports this
+            { label: 'summary', value: 'summary' },
+            { label: 'provided', value: 'frame' },
+          ],
+        },
+        defaultValue: 'none',
+      })
+      .addSelect({
+        category: ['Table footer'],
+        path: 'footerSummary.reducer',
+        name: 'Summarize',
+        description: 'Summarize values on footer',
+        settings: {
+          options: [
+            { value: 'min', label: 'Min' },
+            { value: 'max', label: 'Max' },
+            { value: 'sum', label: 'Sum' },
+            { value: 'avg', label: 'Avg' },
+          ],
+        },
+        defaultValue: 'sum',
+        showIf: (cfg) => cfg.footerMode === 'summary',
+      })
```

Comment:
You can use std stats editor for this maybe: https://github.com/grafana/grafana/blob/53630b5f13095eea309c0560b6b100f6b1b8295c/public/app/plugins/panel/stat/types.ts#L79